### ISSUE TITLE
Grid System

### ISSUE THREAD

**Issue by ateesdalejr** [vutondesign/bronze#7]
Will you consider adding a class based grid system? Maybe something like:
```html
<div class="row">
    <div class="cols5">hey</div>
    <div class="offset5 span2">there</div>
</div>
```
A full row should be worth 12 columns. You could read mdn's article on grid systems if you need help on implementing this.

**Comment by vutondesign** [comment]
Good idea, marking this ``help-wanted`` but I'll take a look at it myself if nobody takes it.

**Comment by vutondesign** [comment]
Fixed in #12
